modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @file
+ * Include basic hooks for the module functionality.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function ding_webtrekk_menu() {
+  $items = [];
+  $items['admin/config/ding/webtrekk'] = [
+    'title' => 'Webtrekk',
+    'description' => 'Settings for the Webtrekk analytics tool.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_webtrekk_admin_settings_form'),
+    'access arguments' => array('administer site configuration'),
+    'file' => 'ding_webtrekk.admin.inc',
+  ];
+  
+  return $items;
+}
+
+/**
+ * Implements hook_page_alter().
+ */
+function ding_webtrekk_page_alter(&$page) {
+  $script = 'window._ti = window._ti || {};';
+
+  $title = drupal_get_title();
+  if ($title) {
+    $script .= "window._ti['pageTitle'] = '$title';";
+  }
+  $domain = variable_get('webtrekk_ti_domain', '');
+  $id = variable_get('webtrekk_ti_id', '');
+
+  $page['page_bottom']['webtrekk']['#attached']['js'] = [
+    [
+      'data' => $script,
+      'type' => 'inline',
+    ],
+    [
+      'data' => 'window._tiConfig = window._tiConfig || {
+        tiDomain: ' . "'$domain'," .
+        'tiId: ' . "'$id'," .
+        'option: {}
+      };
+/** start TagIntegration loader  */
```

Comment:
Could this [part of the file be ignored from analysis](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file)?